aspnetcore/security/data-protection/implementation/key-storage-format.md

```diff
@@ -57,7 +57,7 @@ The particular format of the \<descriptor> element depends on the authenticated
 
 ## The \<encryptedSecret> element
 
-An <encryptedSecret> element which contains the encrypted form of the secret key material may be present if [encryption of secrets at rest is enabled](xref:security/data-protection/implementation/key-encryption-at-rest#data-protection-implementation-key-encryption-at-rest). The attribute decryptorType will be the assembly-qualified name of a type which implements IXmlDecryptor. This type is responsible for reading the inner <encryptedKey> element and decrypting it to recover the original plaintext.
+An <encryptedSecret> element which contains the encrypted form of the secret key material may be present if [encryption of secrets at rest is enabled](xref:security/data-protection/implementation/key-encryption-at-rest). The attribute decryptorType will be the assembly-qualified name of a type which implements IXmlDecryptor. This type is responsible for reading the inner <encryptedKey> element and decrypting it to recover the original plaintext.
```

Comment:
- Code fence <encryptedSecret>, decryptorType, IXmlDecryptor, and <encryptedKey>
- will be --> is

Reply:
This topic (*Key storage format*) needs more work. I opened https://github.com/aspnet/Docs/issues/7758 to address it.